apps/estuary/docker-compose.yml

```diff
@@ -0,0 +1,57 @@
+---
+version: '3.4'
+services:
+  zookeeper:
+    image: wurstmeister/zookeeper
+    ports:
+    - "2181:2181"
+  kafka:
+    image: wurstmeister/kafka
+    depends_on:
+    - zookeeper
+    ports:
+    - "9092:9092"
+    environment:
+    - "KAFKA_ADVERTISED_LISTENERS=INSIDE://:9094,OUTSIDE://${HOST_IP:-localhost}:9092"
+    - "KAFKA_LISTENER_SECURITY_PROTOCOL_MAP=INSIDE:PLAINTEXT,OUTSIDE:PLAINTEXT"
+    - "KAFKA_LISTENERS=INSIDE://:9094,OUTSIDE://:9092"
+    - "KAFKA_INTER_BROKER_LISTENER_NAME=INSIDE"
+    - "KAFKA_CREATE_TOPICS=test-topic:1:1"
```

Comment:
I don't think it's necessary to create this topic when kafka starts. We only care about the `event-stream` topic at this point

Reply:
Unfortunately it seems that the healthcheck for kafka does not succeed unless we create a topic here.  We might be able to get around this but I'm unsure if the JSR is there at this moment